.github/workflows/draw-zmk.yml

```diff
@@ -34,6 +34,11 @@ on:
         default: "[Skip CI] keymap-drawer render"
```

Comment:
```suggestion
        default: "[Automated] keymap-drawer render"
```
Perhaps?

Reply:
Not sure it's needed since GH already adds the "github bot" contributor to the commits, so this only adds noise and takes valuable characters, don't you think?

Reply:
![Screenshot 2023-07-19 at 1 26 03 PM](https://github.com/caksoylar/keymap-drawer/assets/33695/9e1f6242-27f5-4384-b8bc-29f6919dfc9c)
![Screenshot 2023-07-19 at 1 26 24 PM](https://github.com/caksoylar/keymap-drawer/assets/33695/5a853a13-0e03-4754-a2bd-a9f0e6a629f5)

Reply:
Good point, I guess we can just remove the square bracketed part then.